frontend/src/classes/PoolGameAreaController.ts

```diff
@@ -67,6 +73,12 @@ export default class PoolGameAreaController extends (EventEmitter as new () => T
   // Current state of the game that we send to the front end for rendering
   public currentModel: PoolGameAreaModel;
 
+  // A history of the game models at every tick-- index 0 is the model at tick 0, index 1 is the model at tick 1, etc.
+  public modelHistory: PoolGameAreaModel[] = [];
```

Comment:
does the model history only save the history for one move? or more than that? could be specified in the documentation

Reply:
kk will add that rn, good point